electron/remote/get-tls-config.js

```diff
@@ -2,5 +2,5 @@ import path from 'path';
 import ClientApi from 'daedalus-client-api';
 
 export default function getTlsConfig() {
-  return ClientApi.tlsInit(path.join(__dirname, '../../tls/ca.crt'));
+  return ClientApi.tlsInit(path.join(__dirname, '../../tls/ca/ca.crt'));
```

Comment:
I believe it should be `../../../../tls/ca/ca.crt` but lets try with this one.

Reply:
@akegalj im sorry i forgot to delete these files in `remote` folder, we are not using them anymore, so please ignore these paths.

Reply:
I have removed unused files.

Reply:
@akegalj, @DominikGuzei, aha, so we're _still_ going to refer to the CA cert relative to `%INSTALL_DIR%`?

So I need to fix `Launcher.hs` back?

Reply:
@deepfire I wasn't paying attention to the discussion. 

@DominikGuzei and you agreed to have ca.cert at which dir ?